lisa/transformers/kernel_source_installer.py

```diff
@@ -171,17 +214,24 @@ def _modify_code(self, node: Node, code_path: PurePath) -> None:
             self._log.debug(f"modifying code by {modifier.type_name()}")
             modifier.modify()
 
-    def _build_code(self, node: Node, code_path: PurePath) -> None:
+    def _build_code(self, node: Node, code_path: PurePath, config_type: str) -> None:
         self._log.info("building code...")
 
         uname = node.tools[Uname]
         kernel_information = uname.get_linux_information()
 
-        result = node.execute(
-            f"cp /boot/config-{kernel_information.kernel_version_raw} .config",
-            cwd=code_path,
-        )
-        result.assert_exit_code()
+        if self.__is_dom0(node) and config_type == "dom0":
+            result = node.execute(
+                "cp arch/x86/configs/mshv_defconfig .config",
```

Comment:
The path to the config file should be configured from runbook. It will be an optional property and if it's not defined, we will fall back to using the config file from `/boot`.

Reply:
I will change config_type to config_path use it as you described.

Reply:
Done